frontend-ui/src/components/ScheduleActionButton.vue

```diff
@@ -122,7 +122,7 @@
       :disabled="working"
       @click="emit('fire-existing-task')"
     >
-      <v-icon size="small" class="mr-1">mdi-sort-amount-up</v-icon>
+      <v-icon size="small" class="mr-1">mdi-fire</v-icon>
```

Comment:
I'm sorry, I just realize we have the same issue with the "Request with high priority" button:

<img width="333" height="60" alt="image" src="https://github.com/user-attachments/assets/e08d8b61-cdb0-4dfb-8a3b-57a68104de05" />